api_core/google/api_core/datetime_helpers.py

```diff
@@ -169,6 +156,27 @@ def from_rfc3339_nanos(value):
     return bare_seconds.replace(microsecond=micros, tzinfo=pytz.utc)
 
 
+def from_rfc3339_nanos(value):
```

Comment:
I think we can just have `from_rfc3339_nanos=from_rfc3339`. This isn't typically directly used by users, and keeping it around as an alias isn't going to be a maintenance burden.